raiden/ui/cli.py

```diff
@@ -454,7 +454,7 @@ def options(func: Callable) -> Callable:
                     "Mediation fee as ratio of mediated amount in parts-per-million "
                     "(10^-6) for a certain token address."
                 ),
-                type=(ADDRESS_TYPE, click.IntRange(min=0, max=10 ** 6)),
```

Comment:
Can you also edit the argument docs to reflect this limitation?

Reply:
What do you mean?

Reply:
The docstring ... argument docs ... cli help string .. I am not sure how to call it differently :smile: 

It's the string literal on the lines right above.

Reply:
Do you want me to add the limit there?

Reply:
yep. From the help string I would assume the max is 1 million, not `400_000`

Reply:
Ah, the helpstring just describes the unit, which is parts per million, Note the minus in there. But I'll add the limits.

Reply:
Thanks.